pkgs/development/libraries/harfbuzz/default.nix

```
@@ -19,18 +19,19 @@
 , CoreText
 , withCoreText ? false
 , withIcu ? false # recommended by upstream as default, but most don't needed and it's big
-, withGraphite2 ? true # it is small and major distros do include it
+, withGraphite2 ? if stdenv.hostPlatform.isMinGW then false else true # it is small and major distros do include it
```

```suggestion
, withGraphite2 ? lib.meta.availableOn stdenv.hostPlatform graphite2 # it is small and major distros do include it
```
I think this would be a lot cleaner. We maybe need to adjust the meta.platforms of graphite though. Or am I assuming wrong that this is not a problem of graphite2 but in harbuzz itself?

---

Great suggestion. Fixed.